packages/block-editor/src/components/block-list/block.js

```diff
@@ -564,16 +420,13 @@ const applyWithSelect = withSelect(
 			isValid,
 			isSelected,
 			isAncestorOfSelectedBlock,
-			isCapturingDescendantToolbars,
-			hasAncestorCapturingToolbars,
```

Comment:
I don't expect the removed props from this component to have been used by plugins (blockedit filter),  that said, we need to communicate this with a dev note.

Reply:
It was added recently, so it won't be in any WP release. #18440